### ISSUE TITLE
'hide' not being honored

### ISSUE THREAD

**Issue by phillip-haydon** [miromannino/Justified-Gallery#175]
I have no idea how to reproduce this, but 'hide' is not being honored. 

Basically what is happening is when using the following settings:

```
                    $("photos-list.justify-wrapper").justifiedGallery({
                        rowHeight: 220,
                        lastRow: 'hide',
                        waitThumbnailsLoad: false,
                        margins: 4,
                        captions: false
                    });
```

The last row gets added and it calls start analyze and calls `analyzeImages`

This results in the row being being rendered and all hidden images to show. 

It only happens when images are dynamically loaded in. So if I copy the HTML that is rendered (removing all the JG stuff) it renders correctly. The moment the images get dynamically loaded in i.e:

ajax request > use riot to repeat images to the ui > invoke justifyGallery

It all ends up weird and no longer honors the 'hide'. So on initial render I get:

![image](https://cloud.githubusercontent.com/assets/895629/16501330/874d9bbe-3f3b-11e6-967d-7c828ebf7e47.png)

If I resize the UI so that image goes to a full row and then resize it back:

![image](https://cloud.githubusercontent.com/assets/895629/16501354/a2bf07f2-3f3b-11e6-846f-d7d14c37705b.png)

It re-renders but still does not honor 'hide'.

**Comment by phillip-haydon** [comment]
@miromannino I managed to create a sample page on my website that reproduces it. Would you mind if I emailed it to you to share in private, I don't want to share my non-live site on here. If not then I'll abstract it out into something I can share on here.

**Comment by phillip-haydon** [comment]
OH

I can confirm now, it's a bug introduced into 3.6.1, if I change the version to be the same as your demo, 3.6.0, the problem goes away! It works absolutely perfectly with 3.6.0, breaks in 3.6.1.

**Comment by phillip-haydon** [comment]
Bug was fixed in #165 which I guess hasn't been released yet. :(

**Comment by miromannino** [comment]
Yes, should be fixed in 3.6.2
Tell me if in the current version you can download in the master branch it's everything fine.
Thank you!
